lighthouse-core/lib/lh-error.js

```diff
@@ -47,6 +47,8 @@ const UIStrings = {
   internalChromeError: 'An internal Chrome error occurred. Please restart Chrome and try re-running Lighthouse.',
   /** Error message explaining that fetching the resources of the webpage has taken longer than the maximum time. */
   requestContentTimeout: 'Fetching resource content has exceeded the allotted time',
+  /** Error message explaining that the webpage is non-HTML, so audits are ill-defined **/
+  docTypeInvalid: 'The page provided is not HTML',
```

Comment:
maybe we should parameterize this message, so the user knows what to fix? Something like
```suggestion
  docTypeInvalid: 'The page provided is not HTML (served as MIME type {mimeType}).',
```

Reply:
I like that idea, my initial message had the mimeType information but wasn't as easy to read as this, I'll change it!